src/sections/Contact/contact-content.js

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "gatsby";
+
+// Icons
+import { BsThreeDots } from "react-icons/bs";
+
+// Images
+import Jobs_White from "../../images/contact/jobs_white.svg";
+import Support_White from "../../images/contact/support_white.svg";
+import Partner_White from "../../images/contact/partner_white.svg";
+import Contact_White from "../../images/contact/contact_white.svg";
+
+const Wrapper = styled.div`
```

Comment:
Can you please create a separate style file and import the wrapper from there?

Reply:
Sure!